Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/FareProduct.hs

```diff
@@ -44,24 +44,44 @@ findAllFareProductForVariants (Id merchantOpCityId) tripCategory area =
     [ Se.And
         [ Se.Is BeamFP.merchantOperatingCityId $ Se.Eq merchantOpCityId,
           Se.Is BeamFP.area $ Se.Eq area,
-          Se.Is BeamFP.tripCategory $ Se.Eq tripCategory
+          Se.Is BeamFP.tripCategory $ Se.Eq tripCategory,
+          Se.Is BeamFP.timeBounds $ Se.Eq Domain.Unbounded
```

Comment:
Shouldn't we consider bounded in this, what if unbounded fare Product is not there for a tripCategory?